raiden_libs/messages/request_paths.py

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8 -*-
+from typing import Dict
+import struct
+
+import jsonschema
+from eth_utils import is_address, decode_hex
+
+from raiden_libs.messages.message import Message
+from raiden_libs.properties import address_property
+from raiden_libs.messages.json_schema import REQUEST_PATHS_SCHEMA
+from raiden_libs.utils import eth_verify
+from raiden_libs.types import Address
+
+
+class RequestPaths(Message):
+    """ A message to request a path from PFS. It is sent from a raiden node to the PFS. """
+    def __init__(
+        self,
+        token_network_address: Address,
+        target_address: Address,
+        value: int = 1,
```

Comment:
do default values really make sense here? It'd be better to have all required fields enforced.

Reply:
Agreed.